packages/arb-token-bridge-ui/src/state/cctpState.ts

```diff
@@ -117,6 +117,9 @@ function parseTransferToMergedTransaction(
     tokenAddress: getUsdcTokenAddressFromSourceChainId(sourceChainId),
     depositStatus: DepositStatus.CCTP_DEFAULT_STATE,
     isCctp: true,
+    // TODO: Fix those when adding CCTP
```

Comment:
What about this?

Reply:
As discussed, this will be added in the 3rd PR.